src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/IssueCreateDialog.kt

```diff
@@ -72,24 +72,32 @@ class IssueCreateDialog(
      *
      */
     fun showAllFields(): IssueCreateDialog {
-        val configureFieldsDialogId = "inline-dialog-field_picker_popup"
-        val dialogLocator = By.id(configureFieldsDialogId)
+        val configureFieldsDialogClass = "qf-picker"
+        val dialogLocator = By.className(configureFieldsDialogClass)
+
         try {
             openConfigureFieldsDialog(dialogLocator)
         } catch (e: TimeoutException) {
             //we probably sometimes click too fast, but no idea what we should wait for
             openConfigureFieldsDialog(dialogLocator)
         }
-        val locator = By.xpath("//div[@id='$configureFieldsDialogId']//dd[1]//a")
-        if (driver.isElementPresent(locator)) {
-            driver.wait(elementToBeClickable(locator)).click()
-            driver.wait(invisibilityOfElementLocated(dialogLocator))
-            driver.wait(visibilityOfElementLocated(dialogLocator))
+
+        val allFieldsLinkLocator = getVisibleAllFieldsLinkLocator(configureFieldsDialogClass)
+        if (driver.isElementPresent(allFieldsLinkLocator)) {
+            driver.wait(elementToBeClickable(allFieldsLinkLocator)).click()
+            val newAllFieldsLinkLocator = getVisibleAllFieldsLinkLocator(configureFieldsDialogClass)
```

Comment:
Oh snap, we gotta re-find it here? Was the `jira-actions` dev loop sufficient to figure this out?

Reply:
Nope, this is a drive-by fix actually. Turned out that this code, that switches from "Custom" to "All" failed, most likely due to race condition. I discovered it during local testing.